test/jdk/javax/swing/GraphicsConfigNotifier/StalePreferredSize.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The date in the `BasicMenuItemUI.java` can be updated as well.

Reply:
ok